zh-hant/lessons/libraries/bypass.md

```diff
@@ -0,0 +1,276 @@
+---
+version: 1.0.0
```

Comment:
@vbrazo and @MikeRogers0 is there something wrong with our config file? These files should be excluded but they're not nor does it appear our `excluded_words` is working entirely:

https://github.com/elixirschool/elixirschool/blob/master/.typo-ci.yml#L37-L38

Reply:
PR added https://github.com/elixirschool/elixirschool/pull/2135 :)

Reply:
Thank you @MikeRogers0 🎉

Reply:
FYI if you merge in the latest master, it should update & not find any typos :)